index.js

```diff
@@ -0,0 +1,74 @@
+const IncomingWebhook = require('@slack/client').IncomingWebhook;
+const humanizeDuration = require('humanize-duration');
+const config = require('./config.json');
+
+module.exports.webhook = new IncomingWebhook(config.SLACK_WEBHOOK_URL);
+module.exports.status = config.GC_SLACK_STATUS;
+
+// subscribe is the main function called by GCF.
+module.exports.subscribe = (event, callback) => {
+  let build = module.exports.eventToBuild(event.data.data);
```

Comment:
Are we guaranteed that event.data.data always exists? If not, it would be safer to verify beforehand